src/colorizer/Dataset.ts

```diff
@@ -44,10 +68,16 @@ const defaultMetadata: DatasetMetadata = {
   startTimeSeconds: 0,
 };
 
+/** Maps from the feature label to its metadata, including relative filepath, type, and units. */
+type FeatureMap = Record<string, FeatureMetadata>;
+/** Maps from the feature label to its relative filepath only. */
+type DeprecatedFeatureMap = Record<string, string>;
```

Comment:
Nit: If you ever have to do this again, you will have DeprecatedDeprecatedFeatureMap.  This is a reasonable place to have FeatureMapV1, FeatureMapV2, and FeatureMap=FeatureMapV2.  That's how we did it in Simularium viewer.

Another way to deal with this is to convert extremely early, as soon as the json file is read, so that you only ever have to deal with FeatureMap and DeprecatedFeatureMap only need exist for the single conversion function.  It could be cleaner to move it out of here and into its own ts file